cli/actions/datastore.go

```diff
@@ -0,0 +1,147 @@
+package actions
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/kubeshop/tracetest/cli/file"
+	"github.com/kubeshop/tracetest/cli/openapi"
+	"github.com/kubeshop/tracetest/cli/utils"
+	"github.com/mitchellh/mapstructure"
+	"go.uber.org/zap"
+)
+
+type dataStoreActions struct {
+	resourceArgs
+}
+
+var _ ResourceActions = &dataStoreActions{}
+
+func NewDataStoreActions(options ...ResourceArgsOption) *dataStoreActions {
+	args := NewResourceArgs(options...)
+
+	return &dataStoreActions{
+		resourceArgs: args,
+	}
+}
+
+func (d *dataStoreActions) Name() string {
+	return "datastore"
+}
+
+func (d *dataStoreActions) Apply(ctx context.Context, args ApplyArgs) error {
+	if args.File == "" {
+		return fmt.Errorf("you must specify a file to be applied")
+	}
+
+	d.logger.Debug(
+		"applying analytics config",
+		zap.String("file", args.File),
+	)
+
+	fileContent, err := file.ReadRaw(args.File)
+	if err != nil {
+		return fmt.Errorf("could not read file: %w", err)
+	}
+
+	if fileContent.Definition().Type != "DataStore" {
+		return fmt.Errorf(`file must be of type "DataStore"`)
+	}
+
+	var dataStore openapi.DataStore
+	mapstructure.Decode(fileContent.Definition().Spec, &dataStore)
+
+	return d.update(ctx, fileContent, currentConfigID)
+}
+
+func (d *dataStoreActions) List(ctx context.Context, args ListArgs) error {
+	return ErrNotSupportedResourceAction
+}
```

Comment:
I remember we decided on having a `list [resource]` and a `get [resource] --id [id]` for every resource.

Reply:
I believe the idea of not supporting a list for data stores is because there is only one current at a time. Similar to polling profile or config